data/src/main/java/com/microsoft/azure/kusto/data/KustoType.java

```diff
@@ -0,0 +1,55 @@
+package com.microsoft.azure.kusto.data;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.UUID;
+import java.util.function.Function;
+
+/**
+ * Represents various ADX data types and their bindings to Java classes
+ * 
+ * @author alex
+ *
+ * @see https://docs.microsoft.com/azure/data-explorer/kusto/query/scalar-data-types/
+ *
+ * @param <C>
+ */
+public class KustoType<C> {
+	
+	public static final KustoType<Integer>			INTEGER						= new KustoType<>(Integer.class);
```

Comment:
We use spaces instead of tabs (tell me about it :-) )
So besides the left indentation, we suggest using a single space between variables' signature, name and what the variable is set to.

Reply:
Hi @yihezkel 

Like I've already mentioned to to @ohadbitt in this PR, I can fix some of the styling issues here and there, but something tells me that you guys have your own formatter and you'd end up using it on the PR anyway, so I don't see the point in fixing individual formatting issues.
I can change the tabs to spaces, no problem, but then we'd just run into some other formatting issue that goes against your code conventions. Therefore, I think that the better approach would be to format the code using your formatter (either before or after merging the PR), or somehow give me access to your formatter so I can do it myself.